modules/troubleshooting-disabling-autoreboot-mco.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-operator-issues.adoc
+
+[id="troubleshooting-disabling-autoreboot-mco_{context}"]
+= Disabling Machine Config Operator from automatically rebooting
+
+When configuration changes are made by the Machine Config Operator, {op-system-first} must reboot for the changes to take effect. Whether the configuration change is automatic, such as when `kube-apiserver-to-kubelet-signer` CA is rotated, or manual, such as when a registry or SSH key is updated, an {op-system} node reboots automatically unless is is  paused.
+
+To avoid unwanted disruptions, you can modify the machine config pool to prevent automatic rebooting after the Operator makes changes to the machine config.
```

Comment:
It sounds implied but I want to stress that pausing a given pool will pause _everything_ and an unpause would cause anything queued to be applied to start being applied. So, it's really _everything_

Reply:
@runcom: Noted. I added a `NOTE` explaining pausing in more detail in this section, and a separate `NOTE` in the unpausing step below.